filters/all.js

```diff
@@ -107,7 +107,10 @@ module.exports = ({ Nunjucks }) => {
   Nunjucks.addFilter('isArray', (arr) => {
     return Array.isArray(arr);
   });
-
+  
+  Nunjucks.addFilter('additionalPropertiesAllowedForAll', (items) => items.every(p => p.additionalProperties() === true || p.additionalProperties() === undefined || p.additionalProperties() === null));
+  Nunjucks.addFilter('additionalPropertiesDisabledForAll', (items) => items.every(p => p.additionalProperties() === false) );
```

Comment:
I couldn't find a nice way to in-line this within the template. Suggestions welcome!